gatsby-config.js

```diff
@@ -31,10 +32,12 @@ module.exports = {
       options: {
         postCssPlugins: [require('tailwindcss')('./tailwind.config.js')]
       }
-    },
-    {
-      resolve: `gatsby-plugin-purgecss`,
-      options: { tailwind: true }
     }
+    // @TODO: Add gatsby-pluin-purgecss to config
```

Comment:
Instead comment out the whole code, let's create a ticket in asana and let here if you want only a single line like:
`// @TODO: Add gatsby-pluin-purgecss to config`